clients/snowflake/staging.go

```diff
@@ -63,35 +58,31 @@ func (s *Store) PrepareTemporaryTable(tableData *optimization.TableData, tableCo
 		}
 	}()
 
+	// Upload the CSV file to Snowflake
 	if _, err = s.Exec(fmt.Sprintf("PUT file://%s @%s AUTO_COMPRESS=TRUE", fp, addPrefixToTableName(tempTableName, "%"))); err != nil {
 		return fmt.Errorf("failed to run PUT for temporary table: %w", err)
 	}
 
+	// COPY the CSV file (in Snowflake) into a table
 	copyCommand := fmt.Sprintf("COPY INTO %s (%s) FROM (SELECT %s FROM @%s)",
-		// Copy into temporary tables (column ...)
 		tempTableName, strings.Join(tableData.ReadOnlyInMemoryCols().GetColumnsToUpdate(s.config.SharedDestinationConfig.UppercaseEscapedNames, &sql.NameArgs{
 			Escape:   true,
 			DestKind: s.Label(),
 		}), ","),
-		// Escaped columns, TABLE NAME
 		escapeColumns(tableData.ReadOnlyInMemoryCols(), ","), addPrefixToTableName(tempTableName, "%"))
 
 	if additionalSettings.AdditionalCopyClause != "" {
 		copyCommand += " " + additionalSettings.AdditionalCopyClause
 	}
 
-	_, err = s.Exec(copyCommand)
-	if err != nil {
-		return fmt.Errorf("failed to load staging file into temporary table: %w", err)
+	if _, err = s.Exec(copyCommand); err != nil {
+		return fmt.Errorf("failed to run copy into temporary table: %w", err)
 	}
 
 	return nil
 }
 
-// loadTemporaryTable will write the data into /tmp/newTableName.csv
-// This way, another function can call this and then invoke a Snowflake PUT.
-// Returns the file path and potential error
-func (s *Store) loadTemporaryTable(tableData *optimization.TableData, newTableName string) (string, error) {
+func (s *Store) writeTemporaryTable(tableData *optimization.TableData, newTableName string) (string, error) {
```

Comment:
Nit: Maybe `writeTemporaryTableFile` to indicate that we're writing out a file and not writing to a temporary table.

Reply:
Nit: Could also use `os.TempDir()` here.